gateleen-core/src/main/java/org/swisspush/gateleen/core/util/LockUtil.java

```diff
@@ -70,7 +70,7 @@ public static void releaseLock(Lock lockImpl, String lock, String token, Logger
                     log.debug("Released lock '{}' with token '{}'", lock, token);
                 }
             } else {
-                log.error("Could not release lock '{}'. Message: {}", lock, releaseEvent.cause().getMessage());
+                log.error("Could not release lock '{}'.", lock, new Exception("stacktrace", releaseEvent.cause()));
```

Comment:
Can't we simply write

log.error("Could not release lock '{}'.", lock, releaseEvent.cause());

?

Reply:
The stack would be useless if we do so.